docs/user/DockerUsage.md

```diff
@@ -0,0 +1,107 @@
+# Using Docker Container
+
+## Prereqs
+
+- Docker [installed on your workstation](../../README.md#docker).
+
+## Preparation
+
+### Docker image
+
+Run the following command to create the `viya4-deployment` Docker image using the provided [Dockerfile](../../Dockerfile)
+
+```bash
+docker build -t viya4-deployment .
+```
+The Docker image `viya4-deployment` will contain ansible, cloud provider cli's and 'kubectl' executables. The Docker entrypoint for the image is `ansible-playbook` that will be run with sub-commands in the subsequent steps.
+
+### Cloud Authentication
+
+The docker container contains the carious cloud clis for interacting with the various clouds. See [ansible cloud authentication](AnsibleCloudAuthentication.md) when deploying to GCP with external postgres
```

Comment:
"The docker container includes the necessary cloud clis for interacting with the cloud providers."